### PR TITLE
MULTIARCH-4345: Rename to multiarch-tuning-operator

### PR DESCRIPTION
**Pull request by aleskandro**
/wip

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# Multiarch Manager Operator
+# Multiarch Tuning Operator
 
-The Multiarch Manager Operator (MMO) enhances the user experience for administrators of Openshift clusters with
+The Multiarch Tuning Operator (MTO) enhances the user experience for administrators of Openshift clusters with
```
Comment (Prashanth684) [inline README.md]: let's remove the "MTO" acronym here and wherever else

File: bundle/manifests/multiarch-tuning-operator.clusterserviceversion.yaml [hunk 2]
```diff
@@ -45,13 +45,13 @@ spec:
       kind: PodPlacementConfig
       name: podplacementconfigs.multiarch.openshift.io
       version: v1alpha1
-  description: The Multiarch Manager Operator (MMO) enhances the user experience for
+  description: The Multiarch Tuning Operator (MTO) enhances the user experience for
```
Comment (Prashanth684) [inline bundle/manifests/multiarch-tuning-operator.clusterserviceversion.yaml]: remove MTO

File: bundle/manifests/multiarch-tuning-operator.clusterserviceversion.yaml [hunk 3]
```diff
@@ -341,12 +341,12 @@ spec:
   - supported: true
     type: AllNamespaces
   keywords:
-  - multiarch-manager-operator
-  - mmo
+  - multiarch-tuning-operator
+  - mto
```
Comment (Prashanth684) [inline bundle/manifests/multiarch-tuning-operator.clusterserviceversion.yaml]: remove as well

File: config/manifests/bases/multiarch-tuning-operator.clusterserviceversion.yaml [hunk 4]
```diff
@@ -15,13 +15,13 @@ spec:
       kind: PodPlacementConfig
       name: podplacementconfigs.multiarch.openshift.io
       version: v1alpha1
-  description: The Multiarch Manager Operator (MMO) enhances the user experience for
+  description: The Multiarch Tuning Operator (MTO) enhances the user experience for
     administrators of Openshift clusters  with multi-architecture compute nodes or
```
Comment (Prashanth684) [inline config/manifests/bases/multiarch-tuning-operator.clusterserviceversion.yaml:19]: remove

File: config/manifests/bases/multiarch-tuning-operator.clusterserviceversion.yaml [hunk 5]
```diff
@@ -39,12 +39,12 @@ spec:
   - supported: true
     type: AllNamespaces
   keywords:
-  - multiarch-manager-operator
-  - mmo
+  - multiarch-tuning-operator
+  - mto
```
Comment (Prashanth684) [inline config/manifests/bases/multiarch-tuning-operator.clusterserviceversion.yaml]: remove mto

File: README.md [hunk 6]
```diff
@@ -1,6 +1,6 @@
-# Multiarch Manager Operator
+# Multiarch Tuning Operator
 
-The Multiarch Manager Operator (MMO) enhances the user experience for administrators of Openshift clusters with
+The Multiarch Tuning Operator enhances the user experience for administrators of Openshift clusters with
 multi-architecture compute nodes or Site Reliability Engineers willing to migrate from single-arch to multi-arch
 OpenShift. When diverse CPU architectures coexist within a cluster, the MMO operator stands out as a pivotal tool to
```
Comment (Prashanth684) [inline README.md]: some more instances of "MMO" like the one here and others similar to this too

**Comment by Prashanth684** [comment]
i think this is good to go @aleskandro

**Comment by aleskandro** [comment]
yes, it is!

/unhold

**Pull request merged**